js/libs/core.js

```diff
@@ -501,6 +508,24 @@ Fliplet.FormBuilder = (function() {
         component.methods.initDatepicker = component.methods._initDatepicker;
       }
 
+      component.methods._initTimePicker = function() {
+        var $vm = this;
+
+        $vm.$nextTick(function() {
+          var $el = $($vm.$refs.timepicker).timeEntry({
+            show24Hour: true
```

Comment:
In regards to 24 hours: what about US customers? What difference does this setting make when turned on or off?

Reply:
fixed

Reply:
was there any reason for adding it in first place?

Reply:
i forget about 12h type, now fixed